src/adapters/glexchange-polygon-1/index.ts

```diff
@@ -0,0 +1,92 @@
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+import BigNumber from "bignumber.js";
+import moment from "moment";
+import { EventData } from "web3-eth-contract";
+import path from "path";
+import priceSdk from "../../sdk/price";
+import Matic from "../../sdk/matic";
+import symbolSdk from "../../sdk/symbol";
+import { ISaleEntity, ISymbolAPIResponse } from "../../sdk/Interfaces";
+
+class GlExchange {
+    name: string;
+    token: string;
+    protocol: string;
+    block: number;
+    contract: string;
+    events: string[];
+    pathToAbi: string;
+    range: number;
+    chunkSize: number;
+    sdk: any;
+
+    constructor() {
+        this.name = "glexchange-polygon-1";
+        this.protocol = "polygon";
+	this.token = "0xc2132d05d31c914a87c6611c10748aeb04b58e8f";
+        this.block =  34825626;
+        this.contract = "0x8013bf7e0278891FB82b26FbD56628e12FEd112A";
```

Comment:
All EVM addresses has to be lowercased.

Reply:
Lowercased here https://github.com/GuardianLinkDev/nft-sales-adapters/blob/main/src/adapters/glexchange-polygon-1/index.ts#L69

Anyway I have changed from uppercase to lowercase